Added getLocalDiskFlag for SoftLayer_Virtual_Guest

mattcui opened the pull request:
@maximilien Need to add `getLocalDiskFlag` function to address [tracker 126844449](https://www.pivotaltracker.com/n/projects/1344876/stories/126844449), will submit another PR from `bosh-softlayer-cpi`. Please review and merge. Thanks.

maximilien commented on services/softlayer_virtual_guest_test.go:
```diff
@@ -2127,4 +2127,41 @@ var _ = Describe("SoftLayer_Virtual_Guest_Service", func() {
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("local disk for virtual guest instance", func() {
```
change description to: `It("finds local disk flag for local disk ...", ...`

maximilien commented on services/softlayer_virtual_guest_test.go:
```diff
@@ -2127,4 +2127,41 @@ var _ = Describe("SoftLayer_Virtual_Guest_Service", func() {
 	})
+
+	Context("#GetLocalDiskFlag", func() {
+		BeforeEach(func() {
```
Please add some error path tests. That is test to check when `false` is returned as well as `false` with error

maximilien commented:
Please address feedback and add at least one integration test and I will merge. Thx

mattcui commented:
@maximilien I already addressed your comments about unit and integration test, and passed the integration test through the dev branch pipeline. Please review again. Thanks.

mattcui commented:
@maximilien this PR is a little urgent, I need to submit another one from `bosh-softlayer-cpi` after you merge it. And please help update Godeps of `bosh-softlayer-cpi` with this new change in `softlayer-go`. Thanks.

maximilien commented:
OK merging. Best 👍
